### ISSUE TITLE
conda skeleton pypi does not escape quotes

### ISSUE THREAD

**Issue by NiklasRosenstein** [conda/conda-build#3601]
### Actual Behavior

Using `conda skeleton pypi` produces a `meta.yaml` file where fields in the `about` section may contain string values where quotes should have been escaped, but are not.

### Expected Behavior

Quotes in plain text fields should be escaped.

### Steps to Reproduce

`conda skeleton pypi nr.cli=1.2.0`

```yml
[...]

about:
  home: "https://github.com/NiklasRosenstein/python-nr.cli"
  license: "MIT"
  license_family: "MIT"
  license_file: "LICENSE.txt"
  summary: "This package provides the nr command-line tool which then allows you to use any comman registered to the "nr.cli.commands" entrypoint."
  doc_url: ""
  dev_url: ""
```

The resulting YAML file is invalid because the quotes in the `summary` field are not escaped.

##### `Output of conda info`

```

     active environment : local
    active env location : C:\Users\niklas\.conda\envs\local
            shell level : 2
       user config file : C:\Users\niklas\.condarc
 populated config files : C:\Users\niklas\.condarc
          conda version : 4.7.5
    conda-build version : 3.18.7
         python version : 3.7.3.final.0
       virtual packages : __cuda=10.2
       base environment : C:\Users\niklas\Miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/nrosenstein/win-64
                          https://conda.anaconda.org/nrosenstein/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\niklas\Miniconda3\pkgs
                          C:\Users\niklas\.conda\pkgs
                          C:\Users\niklas\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\niklas\Miniconda3\envs
                          C:\Users\niklas\.conda\envs
                          C:\Users\niklas\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.7.5 requests/2.21.0 CPython/3.7.3 Windows/10 Windows/10.0.17763
          administrator : False
             netrc file : None
           offline mode : False
```

**Comment by marcelotrevisani** [comment]
Indeed, it should scape it. Thanks for spotting this and reporting it to us.
I am going to take a look at it a bit further.

**Comment by marcelotrevisani** [comment]
Fixed with #3648
